libnopegl/src/scene.c

```diff
@@ -199,6 +203,37 @@ static int build_nodes_set(struct ngl_scene *s)
     return ret;
 }
 
+static int track_files(struct ngl_scene *s)
+{
+    ngli_darray_init(&s->files, sizeof(char *), 0);
+    ngli_darray_init(&s->files_par, sizeof(uint8_t *), 0);
+
+    const struct ngl_node **nodes = ngli_darray_data(&s->nodes);
+    for (size_t i = 0; i < ngli_darray_count(&s->nodes); i++) {
+        const struct ngl_node *node = nodes[i];
+        const uint8_t *base_ptr = node->opts;
```

Comment:
uintptr_t

Reply:
Same comment, all params `base_ptr` are `u8` at the moment